src/swagger/api.yaml

```diff
@@ -29,7 +29,7 @@
       "name": "Apache-2.0",
       "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
     },
-    "version": "0.11.0"
+    "version": "0.12.0"
```

Comment:
This file may change slightly once we have an official swagger spec version, but I'm only expecting the version to change at this point

Reply:
No diffs with the 0.12.0 release, so this files does not need to change